package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "postinstall": "mkdir -p /tmp/tezster && cp -n ./config.json /tmp/tezster/config.json",
     "postinstall-win": "mkdir -p /tmp/tezster && copy -n ./config.json /tmp/tezster/config.json",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "mocha",
+    "test-with-coverage": "nyc mocha"
```

Comment:
rename it to 'test-coverage'

Reply:
you should cover files only meant for CLI (like tezster.js, tezster-manager.js ,module folder) and exclude files under lib or any other folder.